src/Sylius/Bundle/SupportBundle/Behat/SupportContext.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\SupportBundle\Behat;
+
+use Behat\Gherkin\Node\TableNode;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+
+class SupportContext extends DefaultContext
+{
+    /**
+     * @Given /^there are following support tickets:$/
+     */
+    public function thereAreSupportTickets(TableNode $table)
+    {
+        $manager = $this->getEntityManager();
+        $repository = $this->getRepository('support_ticket');
+        $categoryRepository = $this->getRepository('support_category');
+
+        foreach ($repository->findAll() as $supportTicket) {
+            $manager->remove($supportTicket);
+        }
+
+        $manager->flush();
```

Comment:
Do `$manager->clear();` after flushing. There may be an error if you will try to add support ticket with the same data as previously deleted.